### DISCUSSION TITLE
[QUESTION] how i can determine who if what time is, something happen

### DISCUSSION THREAD

**Discussion by greensci** [CosmosOS/Cosmos#2464] [Q&A]
hi, i need help with with this, how i can determine who  if what time is, this happend

like:

if(actualTime == "8:00 PM")
{
Console.WriteLine("Good Night");
}
if(actualTime == "8:00 AM")
{
Console.WriteLine("Good Morning");
}

the code is only a example of the question for explain it wihout saying it

how i can do that?

**Comment by LoxyMono** [comment]
you can use the RTC to get the time, and get the times between that to display the message as needed

**Comment by MishaTy** [comment]
Use DateTime.Now

**Reply by greensci** [reply]
ok
